static/templates/settings/admin_bot_form.hbs

```diff
@@ -7,11 +7,11 @@
               widget_name="edit_bot_owner"
               list_placeholder=(t 'Filter users')}}
         </div>
-        <div class="name_change_container">
+        <div class="input-group name_change_container">
             <label for="full_name">{{t "Full name" }}</label>
             <input type="text" autocomplete="off" name="full_name" value="{{ full_name }}" />
         </div>
-        <div class="email_change_container">
+        <div class="input-group email_change_container">
```

Comment:
I think you can remove this change, since the `email_change_container` already has enough space below it, and adding more makes it look too spaced out.

Reply:
Removed, thank you!